tests/python/unittest/test_numpy_op.py

```diff
@@ -948,6 +948,85 @@ def hybrid_forward(self, F, x):
             assert same(ret_mx.asnumpy(), ret_np)
 
 
+@with_seed()
```

Comment:
Use 4-space indentations in Python files.

Reply:
fixed all @haojin2

Reply:
As there are new commits in numpy branch, I rebase and force push.